docs/conf.py

```diff
@@ -278,25 +278,16 @@
 latex_elements = {
     'papersize': 'a4',
     'pointsize': '11pt',
-    'figure_align': 'tbp',
+    'figure_align': 'H',
+    'extraclassoptions': 'openany,oneside',
```

Comment:
In a real book, it is rather common to have chapters only start on the right. But I am fine with this change.

Reply:
I know its useful for printing, but IMHO I'm always confused when I scroll through the PDF on my computer and have up to two consecutive blank pages. I'm not sure whether the majority of PDF consumers print this -- my (completely based on thin air) assumption is that they consume it as a digital document rather than printing. But if you think otherwise, lets keep it friendlier for printing.
On an even more subjective level, I find 346 pages (without the blank ones) less intimidating than 390 pages (with the blank ones).

Reply:
We can (later) address that be rendering multiple versions of PDF -- for printing and electronic consumption. I am perfectly happy with this change.